beacon-chain/core/epoch/epoch_processing.go

```diff
@@ -220,25 +220,25 @@ func ProcessJustificationAndFinalization(state *pb.BeaconState, prevAttestedBal
 	bitfield := state.JustificationBitfield
 	// When the 2nd, 3rd and 4th most recent epochs are all justified,
 	// 2nd epoch can finalize the 4th epoch as a source.
-	if oldPrevJustifiedEpoch == currentEpoch-3 && (bitfield>>1)%8 == 7 {
+	if oldPrevJustifiedEpoch+3 == currentEpoch && (bitfield>>1)%8 == 7 {
```

Comment:
Can you make sure the docs in this function are updated? The pseudocode segment doesn't reflect the new changes.